docs/pywbemcligeneraloptions.rst

```diff
@@ -92,8 +90,7 @@ information on each command-group arguments and options):
   can be used to execute pywbemcli commands without access to a real server.
```

Comment:
If the `--mock-server` option has a parameter, it should be explained, ideally with an example.

Reply:
DONE.